ipatests/test_integration/test_ntp_replacement.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+
+
+class TestNTPreplacement(IntegrationTest):
+    """
+    Test NTP Replacement
+    """
+
+    def test_server_install_without_option_N(self):
+        """
+        test to verify ipa-server-install uses default chrony configuration
+        """
+        result = tasks.install_master(self.master, setup_dns=False)
+        assert result.returncode == 0
+        assert "No SRV records of NTP servers found and no NTP server" \
+               " or pool address was provided." in result.stderr_text
+        assert "Using default chrony configuration." in result.stdout_text
+        tasks.uninstall_master(self.master)
+
+    def test_server_install_with_option_N(self):
+        """
+        test to verify ipa-server-install with option -N
+        """
+        result = tasks.install_master(self.master, setup_dns=False,
+                                      extra_args=['-N'])
+        assert result.returncode == 0
+        assert "Using default chrony configuration." not in result.stdout_text
+        tasks.uninstall_master(self.master)
+
+    def test_server_install_with_ntp_server(self):
+        """
+        test to verify ipa-server-install with option --ntp-server
+        """
+        ntp_server = "1.us.pool.ntp.org"
+        expected_msg = "Configuration of chrony was changed by installer."
+        result = tasks.install_master(self.master, setup_dns=False,
+                                      extra_args=[
+                                          '--ntp-server=%s' % ntp_server])
+        assert result.returncode == 0
+        assert expected_msg in result.stderr_text
+        cmd = self.master.run_command(['cat', '/etc/chrony.conf'])
+        assert cmd.returncode == 0
+        assert "%s" % ntp_server in cmd.stdout_text
+        tasks.uninstall_master(self.master)
+
+    def test_server_install_with_ntp_pool(self):
+        """
+        test to verify ipa-server-install with option --ntp-pool
+        """
+        ntp_pool = "us.pool.ntp.org"
```

Comment:
Please don't hardcode the pool to US pool. Use ``pool.ntp.org``

Reply:
Fixed